packages/core/discovery.config.default.js

```diff
@@ -109,6 +109,10 @@ module.exports = {
     data: process.env.CMS_DATA,
   },
 
+  deliveryPromise: {
+    enabled: false,
```

Comment:
In the next tasks there will be more config added, this is the reason for adding an object `deliveryPromise` and not a single boolean `enableDeliveryPromise: false`